src/main/groovy/de.flachnetz.golang/GoPlugin.groovy

```diff
@@ -108,6 +108,12 @@ class GoPlugin implements Plugin<Project> {
                     }
                 }.toSet()
 
+                // canonical import of this project. should be the same as the canonical import of the root project
+                // plus the relative directory of this project to the root project. From this we calculate the
+                // gopath-absolute directory where the package files lives.
+                def projectCanonicalImport = new File(canonicalImport, relativeTo(project.projectDir, baseDir).path)
```

Comment:
we need to get the directory in which we want to call "go build" for sub-projects